tables/de-g1-detailed.ctb

```diff
@@ -1,4 +1,4 @@
-# liblouis: German Grade 1 Braille (with capitals) (unofficial and experimental)
+# liblouis: German Grade 1 Braille (with capitals)
```

Comment:
I think it's still unofficial and experimental.

Reply:
Ok, Ok